source/features/dim-bots.tsx

```diff
@@ -4,8 +4,7 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-// eslint-disable-next-line import/prefer-default-export
-export const botSelectors = [
+const botSelectors = [
```

Comment:
This is not used anywhere else?

Reply:
Nope :)

Reply:
Since https://github.com/sindresorhus/refined-github/pull/3401